integration-cli/docker_cli_images_test.go

```diff
@@ -121,6 +121,76 @@ func (s *DockerSuite) TestImagesFilterLabelWithCommit(c *check.C) {
 	c.Assert(out, check.Equals, imageID)
 }
 
+func (s *DockerSuite) TestImagesFilterSinceAndBefore(c *check.C) {
+	testRequires(c, DaemonIsLinux)
```

Comment:
Because of the build from scratch which is not available for windoS am I right ? :angel: